DirectEmployers/MyJobs
# [PD-2911] renamed templates with bootstrap3 suffix to v2

## **Purpose:** 
Correct a template naming convention faux pas I started 4 months ago.  v2 templates have the suffix "bootstrap3" appended to them, I changed these to "v2".

## **To Test:**
1.  In Django Admin, in the "Home Page Template" drop down, please select the "home_page_listing_v2.html" template. Please also select the "Version 2" Template version.

2.  Please go to www.my.jobs and ensure the v2 template is loaded.  Everything should exactly like before because there are no functional code modification, only name change.

3.  Please repeat steps 1 and 2 above, but with the "home_page_static_header_footer_v2.html" template selected, which should render the base v2 template with no jobs nor facets.
